### ISSUE TITLE
Sunburst chart : Option to show interactive legend containing first level multiple nodes

### ISSUE THREAD

**Issue by amruta30** [highcharts/highcharts#7429]
#### Expected behaviour
It is expected to have an interactive legend for Sunburst chart showing first level multiple nodes.
Refer JS Fiddle: http://jsfiddle.net/kn10kb7L/2/
In the above fiddle, extra series is used to show the first level multiple nodes in the legend and  a legendItemClick event to make it interactive.
Can you provide any property to achieve the above behaviour?
Also, on interaction with legend, Sunburst chart percentage share value in series data label should be updated automatically instead of recalculation in formatter function.
Refer following JS fiddle that shows similar behaviour in Pie chart : http://jsfiddle.net/acovbaye/

#### Actual behaviour
Currently, on setting the showInLegend property as true in plotOptions, shows the series in legend.
http://jsfiddle.net/w43s2bg0/
However, instead of showing the series it is expected to show seperate nodes.

**Comment by pawelfus** [comment]
Hi @amruta30 

Thank you for your suggestion. Maybe something like `level.showInLegend` would be a good idea. What do you think?

**Comment by amruta30** [comment]
Hi @pawelfus 
Thanks for your reply. Yes `legend.showInLegend` will be good solution. Will this option provide interactive legend behaviour and update the percentage share value in Sunburst chart?

**Comment by pawelfus** [comment]
Yes, I think it should behave like in a pie chart.

**Comment by marco-boeck** [comment]
Yes please! This would be a very helpful enhancement to bring it in line with standard capabilities of other Sunburst chart providers.

**Comment by pawelfus** [comment]
Unstale
